activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Allow generated create_table migrations to include or skip timestamps
+    
+    Fixes GH#28706. Now rails g migration create_foo and rails g model Foo have the same behavior for timestamps since they implement the same migration template.
```

Comment:
Now `rails generate migration create_user` and `rails generate model User` have the same behavior since it is not possible to skip timestamp in both.

Reply:
I went with:     
Now `rails generate migration create_user` and `rails generate model User` have the same behavior for timestamps which are included by default and skipped with `--skip-timestamps`.

Let me know if this works. I squashed down to 1 commit.

Reply:
@rafaelfranca I looked at this again and looked at surrounding commits to the CHANGELOG and realized that the additional explanation didn't really add much. I just went with the one-liner and dropped the secondary description. I think it's clearer and more concise that way.